AppleAuthProvider.cs

```diff
@@ -29,6 +30,7 @@ public class AppleAuthProvider
         private static string RedirectURL { get; set; }
         private string State { get; set; }
         private int ExpirationInMinutes { get; set; }
+        private string AndroidPackageIdentifier { get; set; } = string.Empty;
```

Comment:
Wouldn't this always be stirng.Empty since it is not initialized anywhere and it's a private field so it cannot be set from outside.